run_modified.py

```diff
@@ -0,0 +1,51 @@
+import sys
```

Comment:
Remove this file as it is not being used anymore

Reply:
Made changes as suggested.